io/zenoh-links/zenoh-link-tcp/src/unicast.rs

```diff
@@ -164,6 +164,11 @@ impl LinkUnicastTrait for LinkUnicastTcp {
     fn is_streamed(&self) -> bool {
         true
     }
+
+    #[inline(always)]
+    fn get_auth_identifier(&self) -> LinkAuthId {
+        LinkAuthId::default()
```

Comment:
See my comment on `Default` implementation of `LinkAuthId`. `serial` does not have any level of `LinkAuthId`, so it should be stated clear that is `empty`/`none`. By simply looking at this code it's not apparent.